src/v/http/client.cc

```diff
@@ -91,6 +92,9 @@ void client::check() const {
 
 ss::future<client::request_response_t> client::make_request(
   client::request_header&& header, ss::lowres_clock::duration timeout) {
+    if (unlikely(_stopped)) {
+        throw std::runtime_error("client is stopped");
```

Comment:
since the method uses continuations it's better to use `ss::make_exception_future` instead

Reply:
Good catch, thanks! Fixed.